### ISSUE TITLE
One Solr node seems to be down

### ISSUE THREAD

**Issue by hectorcorrea** [pulibrary/pdc_discovery#354]
One of the two Solr nodes for PDC Discovery seems to be down.

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/568286/199257480-fb45873a-1dec-45ab-ac5a-d5f12319b644.png">

**Comment by hectorcorrea** [comment]
I created a new replica in `lib-solr-prod4` but unfortunately the new replica also reports "recovery failed"

![Screen Shot 2022-11-01 at 10 45 11 AM](https://user-images.githubusercontent.com/568286/199261668-1683df71-5966-4a43-b565-93d06a8d54d0.png)

**Comment by hectorcorrea** [comment]
The Solr suggestion option seems to detect that we have a problem with this collection and suggest and action to fix it

![Screen Shot 2022-11-01 at 11 08 42 AM](https://user-images.githubusercontent.com/568286/199267090-58f5bd5c-f01e-473d-a38b-4aa7be738c33.png)

but clicking that button does not fix it. It does add a new replica to the collection but still with the "recovery failed" status and after clicking this button a few times it looks like Solr added a bunch of failed replicas to our collection:

![Screen Shot 2022-11-01 at 11 10 57 AM](https://user-images.githubusercontent.com/568286/199267378-526bfb48-d47f-40b2-aff0-633d41b841b2.png)

**Comment by hectorcorrea** [comment]
We could not get a new replica to work on the existing `pdc-discovery-production` collection. We tried deleting the existing replicas and re-creating them in `lib-solr-prod4/5/6` to no avail.

Instead we created a new Solr collection called `pdc-discovery-production-2` with two replicas and that worked perfectly. We then re-indexed the data by running the reindex rake task with SOLR-URL env pointing to `http://lib-solr8-prod.princeton.edu:8983/solr/pdc-discovery-production-2`. 

Once we confirmed the data was reindexed in `pdc-discovery-production-2` we **deleted** the old `pdc-discovery-production` collection and created an alias so that the new `pdc-discovery-production-2` collection is also known as `pdc-discovery-production`, this was the PDC discovery code and configuration does not need to be updated.
